### DISCUSSION TITLE
force to navigate to the same url/page

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [TanStack/router#297] [Q&A] [answered]
Hi. Is there a way to force router to navigate to the same page? I want to use this to refresh page when error happens. currently I am using useNavigate to navigate to the same page. But it does nothing.

**Comment by MadadiSaeed** [comment, marked as answer]
Never mind. I found better way.

**Reply by Jared-Dahlke** [reply]
what did you do? window.location.reload()?
